lib/src/clojure_lsp/refactor/transform.clj

```diff
@@ -433,15 +463,13 @@
                                         (mapv (comp symbol name :name)))
               expr-edit            (-> (z/of-string "")
                                        (z/replace `(~fn-sym ~@used-syms)))
-              defn-edit            (-> (z/of-string "\n(defn)\n")
-                                       (z/append-child fn-sym)
-                                       (z/append-child used-syms)
-                                       (z/append-child* (n/newlines 1))
-                                       (z/append-child* (n/spaces (+ defn-col 1)))
-                                       (z/append-child expr-node)
-                                       z/up)]
-          [{:loc   defn-edit
-            :range defn-range}
+              private? false ;; TODO: shouldn't this be true?
```

Comment:
I wish that functions extracted with `extract-function` were private, since what I use it for most often is to create private helper functions. If that'd be an acceptable behavior change, I can submit a follow-up PR. I don't want to change it as part of this PR, because it's separate from `promote-fn`.

Reply:
Sounds like a good idea to me